{"name": "test_negative_create_new_task[invalid_data2-401-You do not have access to this task-ACCESS_083]", "status": "passed", "description": "Create task with wrong name, parent ID, link ID and validate that errors are equal to expected", "steps": [{"name": "Create new task with body: {'name': 'reality', 'links_to': 414}, expected status code: 401", "status": "passed", "steps": [{"name": "Create url with base url and endpoint: /list/901510919856/task\nResult URL: https://api.clickup.com/api/v2/list/901510919856/task", "status": "passed", "start": 1746984965812, "stop": 1746984965812}, {"name": "Send POST request to https://api.clickup.com/api/v2/list/901510919856/task with json={'name': 'reality', 'links_to': 414} and data=None, expected status code 401", "status": "passed", "start": 1746984965812, "stop": 1746984966313}], "start": 1746984965812, "stop": 1746984966313}], "parameters": [{"name": "invalid_data", "value": "{'name': 'reality', 'links_to': 414}"}, {"name": "expected_status_code", "value": "401"}, {"name": "expected_err_message", "value": "'You do not have access to this task'"}, {"name": "expected_err_code", "value": "'ACCESS_083'"}], "start": 1746984965812, "stop": 1746984966313, "uuid": "a2555c36-501b-443e-90a6-7a3d32b5c42a", "historyId": "cdf8e273afcdc2e4ff2f0b38abeba522", "testCaseId": "2b634744f349a3d06f3aee13d601988d", "fullName": "tests.api.test_tasks.TestTasks#test_negative_create_new_task", "labels": [{"name": "feature", "value": "Manage tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "apachai"}, {"name": "thread", "value": "16192-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}